engine/release-notes.md

```diff
@@ -82,6 +82,41 @@ compatibility reasons.
 
 * Fix jsonfile logger: follow logs stuck when `max-size` is set and `max-file=1`. [docker/engine#378](https://github.com/docker/engine/pull/378)
 
+### Known Issues
+
+#### New
+
+* `DOCKER-USER` iptables chain is missing: [docker/for-linux#810](https://github.com/docker/for-linux/issues/810).
+  Users cannot perform additional container network traffic filtering on top of
+  this iptables chain. You are not affected by this issue if you are not
+  customizing iptable chains on top of `DOCKER-USER`.
+     - **Workaround:** Insert the iptables chain after the docker daemon starts.
+       For example:
+       ```
+       iptables -N DOCKER-USER
+       iptables -I FORWARD -j DOCKER-USER
+       iptables -A DOCKER-USER -j RETURN
+       ```
+
+#### Existing
+
+* In some circumstances, in large clusters, docker information might, as part of the Swarm section,
```

Comment:
Maybe update? In some circumstances with large clusters, docker information might include the following error in the Swarm section.